docs/contracts/container.rst

```diff
@@ -1146,14 +1146,14 @@ setContainerShareConfigs
 
   container.setContainerShareConfigs(newConfigs, originalConfigs);
 
-Takes a full share configuration for a accountId (or a list of them), share newly added properties and unshare removed properties from the container. Also accepts a list / instance of the original sharing configurations duplicated loading can be avoided.
+Takes a full share configuration for an accountId (or a list of them), share newly added properties and unshare removed properties from the container. Also accepts a list / instance of the original sharing configurations duplicated loading can be avoided.
 
 ----------
 Parameters
 ----------
 
-#. ``newConfigs`` - :ref:`container_ContainerShareConfig` / :ref:`container_ContainerShareConfig`[]: sharing configurations that should be persisted
-#. ``unshareConfigs`` - :ref:`container_ContainerShareConfig` / :ref:`container_ContainerShareConfig`[]: pass original share configurations, for that the sharing delta should be built (reduces load time)
+#. ``newConfigs`` - :ref:`container_ContainerShareConfig` / :ref:`container_ContainerShareConfig` []: sharing configurations that should be persisted
+#. ``unshareConfigs`` - :ref:`container_ContainerShareConfig` / :ref:`container_ContainerShareConfig` []: pass original share configurations, for that the sharing delta should be built (reduces load time)
```

Comment:
why is it called unshareConfigs when it's the origonal share config ❓bit confusing ...

Reply:
Was the parameter naming, fixed with commit [58107b2b50fba386478fae7dc208209ab40bd7f5](https://github.com/evannetwork/api-blockchain-core/pull/38/commits/58107b2b50fba386478fae7dc208209ab40bd7f5)